src/components/editor-content/editor-content.tsx

```diff
@@ -41,17 +149,199 @@ const EditorContent: React.FC<Props> = () => {
         </Typography>
         <Typography
           variant="h5"
-          style={{
+          style={ {
             marginLeft: theme.spacing(1),
             fontStyle: "italic"
-          }}
+          } }
         >
           { TimeUtils.minuteToHourString(value) }
         </Typography>
       </>
     );
   }
 
+  /**
+   * Renders selector of filter scope
+   */
+  const renderSelectScope = () => {
+    return (
+      <>
+        <FormControl variant="outlined" className={ classes.selectScope }>
+          <TextField
+            select
+            id="scope-select-outlined"
+            size="small"
+            value={ scope }
+            onChange={ handleDateFormatChange }
+          >
+            { renderSelectOptions }
+          </TextField>
+        </FormControl>
+      </>
+    );
+  }
+
+  /**
+   * Renders start date picker 
+   * 
+   * @returns start date picker
+   */
+  const renderStartDatePicker = () => {
+    const { filterStartingDate } = strings.editorContent;
+
+    return (
+      <MuiPickersUtilsProvider utils={ DateFnsUtils } >
+        <Grid className={ classes.timeFilter }>
+          <KeyboardDatePicker
+            variant="inline"
+            views={ [ scope ] }
+            format={ dateFormat }
+            id="date-picker-start"
+            label={ filterStartingDate }
+            value={ selectedStartingDate }
+            onChange={ handleStartDateChange }
+            KeyboardButtonProps={ {"aria-label": `${ filterStartingDate }`} }
```

Comment:
Check that you don't have some kind of automatic formatting enabled in your VSCode settings
```suggestion
            KeyboardButtonProps={{ "aria-label": `${ filterStartingDate }` }}
```